index.html

```diff
@@ -22,10 +22,155 @@ <h1 class="profileName">Harshita Jain</h1>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+        <div class="project">
+            <img class="pr-img" src="images/Rectangle 21 (3).png">
+            <div class="project-ins">
+                <h3>Project name goes here</h3>
+                <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+        <div class="project">
+            <img class="pr-img" src="images/Rectangle 21 (4).png">
+            <div class="project-ins">
+                <h3>Project name goes here</h3>
+                <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+        <div class="project">
+            <img class="pr-img" src="images/Rectangle 21 (5).png">
+            <div class="project-ins">
+                <h3>Project name goes here</h3>
+                <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+       </div>
+    </section>
+    <section class="about">
+        <h2>About</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        <button type="button">
+            Get my CV
+            <img src="images/ic_download.svg">
+        </button>
+        <div class="languages">
+            <h4 class="set-h">Languages</h4>
+            <ul class="set-ul">
+                <li>JavaScript(ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
```

Comment:
- [ ] According to the requirements and the original design, You must stick to the Figma design as much as possible(e.g., fontsm paddings, margins, ..), So please work on this section 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198962716-a126dcd0-5224-4ee5-8fe6-ad53c865e699.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198963043-ae652a9a-819b-4f90-a638-72b0688570b4.png" width="500" /> |


> I suggest removing the `<center>` HTML tag 💡